components/automate-ui/src/app/modules/infra-proxy/data-bags-list/data-bags-list.component.ts

```diff
@@ -42,12 +45,24 @@ export class DataBagsListComponent implements OnInit {
     combineLatest([
       this.store.select(getAllDatabagsForOrgStatus),
       this.store.select(allDataBags)
-    ]).pipe(
-      filter(([getDataBagsSt, allDataBagsState]) =>
-      getDataBagsSt === EntityStatus.loadingSuccess && !isNil(allDataBagsState))
-    ).subscribe(([ _getDataBagsSt, allDataBagsState]) => {
-      this.dataBags = allDataBagsState;
-      this.dataBagsListLoading = false;
+    ]).pipe(takeUntil(this.isDestroyed$))
+    .subscribe(([ getDataBagsSt, allDataBagsState]) => {
+      if (getDataBagsSt === EntityStatus.loadingSuccess && !isNil(allDataBagsState)) {
+        this.dataBags = allDataBagsState;
+        this.dataBagsListLoading = false;
+      } else if (getDataBagsSt === EntityStatus.loadingFailure) {
+        this.dataBagsListLoading = false;
+        this.authFailure = true;
+      }
     });
   }
+
+  ResetKeyTabRedirection(resetLink: boolean) {
```

Comment:
We follow the typical JS convention of having function and method names as `camelCase`.  Class names and decorators do use `PascalCase`.